docs/source/design/ui_design.rst

```diff
@@ -0,0 +1,151 @@
+the latter is more relavant to our design, we show below how to run an SCF
+calculation for a hydrogen molecule using the PsiAPI mode.
+
+.. code-block:: python
+
+    import psi4
+    mol = psi4.geometry('H 0. 0. 0. \n H 0. 0. 1.')
+    energy= psi4.energy('scf/sto-3g')    
+
+Here, mol is the molecule object, which is created using the ``psi4.geometry()``
+function and the energy is computed using the ``psi4.energy()`` function, which
+takes the method name as the required argument. Note that the user do not need
+to pass the molecule object explicitly to the subsequent energy calculation. By
+default energy is computed for the last molecule defined with the
+``psi4.geometry()`` function. Additional options can be set with the
+``psi4.set_options()`` function, which takes a Python dictionary as the required
+argument. 
+
+
+QCEngine
+========
+QCEngine is a general purpose quantum chemistry program interface. It is a
+Python library that provides a common API for quantum chemistry programs.
+QCEngine is an open-source package distributed under the Apache License 2.0.
+
+Below you can find how to run an SCF calculation for a hydrogen molecule using
+QCEngine.
+
+.. code-block:: python
+
+    import qcengine as qcng
+    import qcelemental as qcel
+    mol = qcel.models.Molecule.from_data('H 0. 0. 0. \n H 0. 0. 1.')
+    out = qcng.compute({"molecule": mol, "driver": "energy", "model": {"method": "SCF", "basis": "sto-3g"}})
+    energy = out.return_result
+
+Here, mol is the molecule object (type
+``qcelemental.models.molecule.Molecule``), which is created using the
+``qcel.models.Molecule.from_data()`` function from QCElemental package. The SCF
+energy is computed using the ``qcng.compute()`` function, from QCEngine
+package. Note that, the input for the `qcng.compute` function is a Python
+dictionary with a schema defined by QCElemental.
+
+
+NWChemEx
+========
+
+In NWChemEx, running an SCF calculation for a hydrogen molecule is made easy
+through the use of a Python function with named arguments, as shown below.
+
+.. code-block:: python
+
+    import nwchemex as nwx 
+    energy = nwx.calculate_scf_energy(molecule = 'H 0. 0. 0. \n H 0. 0. 1.', basis = 'sto-3g')
+
+While the example shows two options for the ``nwx.calculate_scf_energy()``
+function, the whole function signature is given below: 
+
+.. code-block:: python
+
+    def calculate_scf_energy(molecule: Union[str, chemist.Molecule], basis:
+    Union[str, simde.type.ao_space], spin: int = 1, max_iterations: int = 50,
+    thresh: float = 1e-10, module_manager: PlugingPlay.ModuleManager = None, 
+    options: Union[Dict, dataclass] = None, **kwargs) -> float:
+
+In this function, the only required argument is ``molecule``, which can either
+be a Python ``string`` or a ``chemist.Molecule`` object. Optional ``basis``
+argument can either be a a Python ``string`` or a ``simde.type.ao_space``, with
+the minimal basis set "sto-3g" chosen as the default value. Additional SCF
```

Comment:
I dislike the idea of defaulting the basis set, especially to such a bad one.

Reply:
Removed the default basis set with [5991da1](https://github.com/NWChemEx-Project/.github/pull/108/commits/5991da1bb2194b8ce2962b414bafa56c03aabe7a).